include/aspect/postprocess/geoid.h

```diff
@@ -94,7 +94,7 @@ namespace aspect
         /**
          * A parameter to control whether to output the spherical harmonic coefficients of the surface dynamic topography contribution
```

Comment:
Seems like you almost caught all the 'dynamic's - here's the only one I found that slipped through.